trk/src/Components/ClimbItem.js

```diff
@@ -61,7 +61,7 @@ const ClimbItem = ({ climb, tapId, fromHome = false, tapTimestamp }) => {
     else {
         return (
             <TouchableOpacity onPress={role === 'climber' ? navigateToDetail : navigateToSet}>
-                <ListItemContainer dotStyle={styles.climbDot}>
+                <ListItemContainer dotStyle={styles.climbDot} grade={climb.grade}>
```

Comment:
I'd remove the climbDot entirely because some gyms still use **colors** in their grading system.

Reply:
But that's for another PR :)

Reply:
Yupp that's what we discussed, but it just looked odd without some background color XD. Kept the red for now, but we can always change it!

Reply:
hehe agreed.
![Screenshot 2024-01-08 at 12 51 57 pm](https://github.com/nagimonyc/trk-app/assets/66965648/632e5028-da65-4b16-9c83-edca7fd5f080)

Just a thought, placing the grade in a square is a design that we had in mind a while back